exercises/practice/game-of-life/.meta/Example.fs

```diff
@@ -0,0 +1,27 @@
+module GameOfLife
+
+let tick (input: int list list) =
```

Comment:
What do you say about have `input` be a multidimensional array? I think this would be a fun exercise to have students work with multidimensional arrays and the [Array2D module](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-collections-array2dmodule.html).

Reply:
The main thing I didn't like about `Array2D` originally is that it's mutable, but maybe that's not a problem.